src/app/ReadClient.cpp

```diff
@@ -22,15 +22,57 @@
  *
  */
 
-#include "lib/core/CHIPTLVTypes.h"
 #include <app/AppBuildConfig.h>
 #include <app/InteractionModelEngine.h>
 #include <app/ReadClient.h>
 #include <app/StatusResponse.h>
+#include <lib/core/CHIPTLVTypes.h>
+#include <lib/support/FibonacciUtils.h>
 
 namespace chip {
 namespace app {
 
+/**
+ * @brief The default resubscibe policy will pick a random timeslot
+ * with millisecond resolution over an ever increasing window,
+ * following a fibonacci sequence upto CHIP_RESUBSCRIBE_MAX_FIBONACCI_STEP_INDEX.
```

Comment:
Fibonacci utils should document what the biggest input index they support (without overflowing) is.

Reply:
updated, the default is 14.